setup.sh

```diff
@@ -74,3 +74,27 @@ if [[ ${PROJECT_TYPE} == 'nodejs' ]]; then
   echo "install npm modules ${NPM_MODULES[@]}"
   npm i ${NPM_MODULES[@]} --save-dev
 fi
+
+read -p "Use default folder structure? [y, N]: " DEFAULT_FOLDER_STRUCTURE </dev/tty
+
+if [[ ${DEFAULT_FOLDER_STRUCTURE} == 'y' ]]; then
+
+  mkdir -p app/{jade/layouts,styles,scripts}
+
+  echo "Created folder structure:"
+  find app -type d -print
+
+  fetch "jade-default"
```

Comment:
Can we have a second parameter to the `fetch()` function with the target? Would save one step.